test/extended/testdata/bindata.go

```diff
@@ -10646,7 +10648,7 @@ objects:
           ports:
           - name: port
             port: 1234
-      - apiVersion: v1
+      - apiVersion: route.openshift.io/v1
```

Comment:
we still want a v1 openshift object that would need to be submitted to /oapi (service is kube)

Reply:
and a groupified openshift object